content/applications/sales/crm/performance/expected_revenue_report.rst

```diff
@@ -0,0 +1,200 @@
+   :align: center
+   :alt: Close up of the expected closing date on a lead in the CRM app.
+
+By pulling a monthly expected revenue report, sales managers can see which team members are reaching
+their goals, and who may need additional assistance to close valuable deals.
+
+Create an expected revenue report
+=================================
+
+To create an expected revenue report, first navigate to :menuselection:`CRM app --> Reporting -->
+Pipeline`. This opens the :guilabel:`Pipeline Analysis` dashboard.
+
+.. important::
+   The *Pipeline Analysis* dashboard includes several filters in the search bar by default. Remove
+   these before adding any additional custom filters.
+
+On the top-left of the report, click :guilabel:`Measures`, then select :guilabel:`Expected Revenue`
+from the drop-down menu.
+
+At the top of the page, click the :guilabel:`🔻(triangle pointed down)` icon to the right of the
+:guilabel:`Search...` bar to open the drop-down menu that contains :guilabel:`Filters`,
+:guilabel:`Group By`, and :guilabel:`Favorites` columns. Under the :guilabel:`Filters` column, click
+:guilabel:`Add Custom Filter`, which opens an :guilabel:`Add Custom Filter` pop-up window.
+
+.. _expected_revenue_report/custom-filters:
+
+Add custom filters
+------------------
+
+In order to generate an expected revenue report, filters need to be created for the following
+conditions:
+
+ - :ref:`Expected closing date <expected_revenue_report/closing-date>`: limits results to only
+   include leads expected to close within a specific time frame.
+ - :ref:`Exclude unassigned leads <expected_revenue_report/unassigned-leads>`: excludes leads
+   without an assigned salesperson.
+ - :ref:`Specific sales teams <expected_revenue_report/sales-team>`: limits results to only include
+   leads assigned to one or more sales teams. This filter is optional and should not be included if
+   the report is intended for the entire company.
+
+.. _expected_revenue_report/closing-date:
+
+Add filter for expected closing date
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On the :guilabel:`Add Custom Filter` pop-up window, click into the first field of the new rule.
+Type `Expected Closing` into the :guilabel:`Search...` bar, or scroll to select it from the list.
+Click in the second field and select :guilabel:`is set`. This limits the results to only include
+leads where an estimated closing date is listed.
+
+Next, click the :guilabel:`➕ (plus)` icon to the right of the rule to duplicate it.
+
+.. tip::
+   Using the :guilabel:`➕ (plus)` icon makes it easy to add multiple rules based on the same
+   filter.
+
+In the second field of the new rule, select :guilabel:`is between` from the drop-down menu. This
+creates a set time frame during which the expected closing date must occur for leads to be included
+in the results.
+
+Click in the date fields, and use the calendar drop-down to add a start and end date. This is
```

Comment:
- this element is called a *popover* window in web design.
- added some more nuanced instruction here to make it super clear 😉 

```suggestion
Click in each date field, one at a time, and use the calendar popover window to add both a start and
end date to the rule. This is
```